src/IISIntegration/src/AspNetCoreModuleV2/RequestHandlerLib/environmentvariablehelpers.h

```diff
@@ -240,6 +242,25 @@ class ENVIRONMENT_VAR_HELPERS
             goto Finished;
         }
 
+        if (pHttpsPort != nullptr)
```

Comment:
GetHttpsPort never returns null, only empty string. If there isn't a binding, or there are two ambiguous bindings, we should let the dev set the env var for us.

Actually, I wonder if we should ever replace the env var if the dev has already defined it. What would happen if they set this env var in the web.config?

Reply:
I'll change it to prefer externally set `ASPNETCORE_HTTPS_PORT`